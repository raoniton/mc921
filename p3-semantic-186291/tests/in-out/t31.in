class Main {
    public int f(int m, int n) {
        if (!(n >= 2)) {
            n = n - 2;
            return n;
        }
        else {
            m = m - 1;
            return m;
        }
    }

    public static void main(String[] args) {
        int m = -9;
        char n = '4';
        print(this.f(m, n));
    }
}