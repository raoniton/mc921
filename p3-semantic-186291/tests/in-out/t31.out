SemanticError: Type mismatch with parameter n @ 16:15
